.github/workflows/CI-MATs.yml

```
@@ -0,0 +1,117 @@
+      actions: read
+      pull-requests: read
+      checks: write
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ github.token }}
+    outputs:
+      should_run: ${{ steps.check.outputs.should_run }}
+      build_environment: ${{ steps.get_env.outputs.environment }}
+    steps:
+      - id: check
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:29: Double quote to prevent globbing and word splitting

(shellcheck)